int8 drone_id
crazyflie_trajectory_manager/VisualizeTrajectoryPayload payload
---
bool proxy_success
bool result
